### ISSUE TITLE
Broken parameters passing in dotnet CLI 7.0.*

### ISSUE THREAD

**Issue by DarkWanderer** [dotnet/sdk#29396]
### Describe the bug
When passing multiple parameters to `dotnet test` with `/p` or `-p` flag, only the first parameter is taken into account

### To Reproduce
Take any repo with test available

Preparation:
`dotnet add <test project file> package coverlet.msbuild`

**Scenario 1**
`dotnet test -p:CollectCoverage=true -p:CoverletOutputFormat=cobertura`

Observed:
* ` Generating report '<path>\coverage.json'`
* Table with coverage summary

Expected:
` Generating report '<path>\coverage.xml'`

**Scenario 2**
`dotnet test -p:CoverletOutputFormat=cobertura -p:CollectCoverage=true`

Observed:
* No coverage generated

Expected:
* Table with coverage
* Table with coverage summary

### Further technical details
- Reproduces in GitHub Actions
- Reproduces locally with Visual Studio 2022 Community Edition installed

**Comment by MarcoRossignoli** [comment]
The issue it's been fixed into 7.0.101 can you try with it? https://dotnet.microsoft.com/en-us/download/dotnet/7.0

**Comment by DarkWanderer** [comment]
Yes, I confirm this [was fixed](https://github.com/DarkWanderer/ClickHouse.Client/actions/runs/3749209826/jobs/6367432497#step:10:7) in 7.0.101, thanks
